src/codegen/generators/variable_generator.rs

```diff
@@ -164,6 +165,18 @@ impl<'ctx, 'b> VariableGenerator<'ctx, 'b> {
             }
         }
 
+        let section = section_mangler::SectionMangler::variable(
+            global_variable.get_name(),
+            // FIXME: Can we unwrap here?
```

Comment:
> In general it should be save to assume that the types will be found in the index at such a late phase. Nevertheless we propagate such Result-errors already in Line 117 so maybe you can also just use the `?`-operator?

Sure thing! will do.

> Do you prefere panics in such situations?

This is what I usually do, but I think it's personal preference so I'm completely fine with propagating the error. As you said, I had assumed that this late in the pipeline we'd *need* to have these types present, and in case they were not present this was a compiler bug which should cause a crash